Translations/bg.po

```diff
@@ -6554,41 +6581,44 @@ msgid ""
 
 #. TRANSLATORS: Quest dialog spoken by Cain
 #: Source/textdat.cpp:105
 msgid "A book that speaks of a chamber of human bones? Well, a Chamber of Bone is mentioned in certain archaic writings that I studied in the libraries of the East. These tomes inferred that when the Lords of the underworld desired to protect great treasures, they would create domains where those who died in the attempt to steal that treasure would be forever bound to defend it. A twisted, but strangely fitting, end?"
-msgstr ""
+msgstr "Книга, в която се говори за Камера с човешки кости? Така, за камера от кости се споменава в някои архаични писания, които изучавах в библиотеките на Изтока. В тези съчинения се казваше, че когато Властелините на подземния свят искали да защитят големи съкровища, те създавали територии, в които онези, които загинали в опит да откраднат съкровището, били завинаги осъдени да го защитават като немъртви. Извратен, но странно подходящ край?"
 
 #. TRANSLATORS: Quest dialog spoken by Ogden
 #: Source/textdat.cpp:107
 msgid "I am afraid that I don't know anything about that, good master. Cain has many books that may be of some help."
-msgstr ""
+msgstr "Страхувам се, че не знам нищо за това, добри господарю. Кайн има много книги, които могат да ви помогнат."
 
 #. TRANSLATORS: Quest dialog spoken by Pipin
 #: Source/textdat.cpp:109
 msgid "This sounds like a very dangerous place. If you venture there, please take great care."
-msgstr ""
+msgstr "Това звучи като много опасно място. Ако се отправяш натам, моля, бъди много внимателен."
 
 #. TRANSLATORS: Quest dialog spoken by Gillian
 #: Source/textdat.cpp:111
 msgid "I am afraid that I haven't heard anything about that. Perhaps Cain the Storyteller could be of some help."
-msgstr ""
+msgstr "Страхувам се, че не съм чувал нищо за това. Може би Кайн Разказвачът би могъл да ти помогне."
 
 #. TRANSLATORS: Quest dialog spoken by Griswold
 #: Source/textdat.cpp:113
 msgid "I know nothing of this place, but you may try asking Cain. He talks about many things, and it would not surprise me if he had some answers to your question."
-msgstr ""
+msgstr "Не знам нищо за това място, но може да попиташ Кайн. Той знае за много неща и няма да се изненадам, ако може да отговори на въпроса ти."
 
 #. TRANSLATORS: Quest dialog spoken by Farnham
 #: Source/textdat.cpp:115
 msgid "Okay, so listen. There's this chamber of wood, see. And his wife, you know - her - tells the tree... cause you gotta wait. Then I says, that might work against him, but if you think I'm gonna PAY for this... you... uh... yeah."
-msgstr ""
+msgstr "Добре, слушай сега. Има тази камера от дърво, сещаш се. И жена му, нали знаеш - нейната - казва на дървото... защото трябва да почакаш. Тогава аз казвам, че това може и да работи срещу него, но ако мислиш, че аз ще ПЛАЩАМ за това... ти... ъъъ... да."
```

Comment:
Due to the complexity of the sentence, I'd try to think of a different way of interpreting things around here. Something like... 

"Добре, слушай сега. За онази Камера на Дървото (bit confusing here regarding the meaning, although since the dialogue continues discussing a tree, I would say it's "Камера на Дървото"), сещаш се. И жена му, нали знаеш, тя,  рече на дървото... защото трябва да почакаш (jeez, the confusion - is this Farnham presuming you have to wait or the person he speaks saying it but Farnham interprets?!). Тогава аз рекох, че това може и да проработи срещу него, но си мислиш, че аз ще ПЛАЩАМ за това... ти... ъ-ъ-ъ... да."

Reply:
The conversation is confusing on purpose. Chamber of wood, is pronounced a bit like a name. Hence the reference of a his wife. The dialog shouldn't make any sense from the beginning till the end.

Reply:
```suggestion
msgstr "Добре, слушай сега. Има тази камера от дърво, сещаш се. И жена му, нали знаеш - нейната - казва на дървото... защото трябва да почакаш. Тогава аз рекох, че това може и да проработи срещу него, но ако мислиш, че аз ще ПЛАЩАМ за това... ти... ъъъ... да."
```